main.tex

```diff
@@ -1263,36 +1366,36 @@ \section{Conclusion}
 % Renamed and edited by Adrian PW
 
 The development of the \astropypkg package and cultivation of the \astropy
-ecosystem is still maintaining significant growth while improving stability,
+ecosystem are still maintaining significant growth while improving in stability,
 breadth, and reliability.
 As the \astropypkg core package becomes more mature, several subpackages have
 reached stability with a rich set of features that help astronomers worldwide to
-perform many daily tasks such as planning observations, analyzing data or
+perform many daily tasks, such as planning observations, analyzing data or
 simulation results, and writing publications.
-The strong emphasis that the \astropy project puts on reliability and code
-correctness helps users to trust the calculations performed with \astropypkg and
+The strong emphasis that the \astropy project puts on reliability and high coding
+standards helps users to trust the calculations performed with \astropypkg and
 to publish reproducible results.
-At the same time, the \astropy ecosystem and core package are growing: new
-functionality is still being contributed, and new affiliated packages are being
+At the same time, the \astropy ecosystem and core package are both growing: new
+functionality is still being contributed and new affiliated packages are being
 developed to support more specialized needs.
 
 The \astropy project is also spreading awareness of best practices in
 community-driven software development.
 This is important as most practicing astronomers were not explicitly taught
 computer science and software development, despite the fact that a major
-fraction of nearly every astronomer's workload today is related to software use
+fraction of many astronomers' workload today is related to software use
```

Comment:
Note: "nearly every" sounds too grave to me unless we have published stats to back that up.